user/views.py

```diff
@@ -12,7 +23,7 @@ def signup_view(request):
     else:
         session_form_data = request.session.get('info_form_data')
         form = SignupForm(session_form_data)
-    return render(request, 'user/signup.html', {'form':form})
+    return render(request, 'user/signup/signup.html', {'form':form})
```

Comment:
辞書のコロンの後は空白入れましょう
以下同様
```suggestion
    return render(request, 'user/signup/signup.html', {'form': form})
```